pynbody/halo/ahf.py

```diff
@@ -52,7 +52,11 @@ def __init__(self, sim, make_grp=None, get_all_parts=None, use_iord=None, ahf_ba
         """
 
         import os.path
-        if not self._can_load(sim):
+        if ahf_basename is not None:
+            can_load_ahf = self._can_load(sim, ahf_basename)
+        else:
+            can_load_ahf = self._can_load(sim)
+        if not can_load_ahf:
```

Comment:
Since `_can_load` already checks for `None`, can you not simplify these 5 lines to `if not self._can_load(sim, ahf_basename)`?

Reply:
indeed. That was stupid double checking.